pkgs/tools/graphics/pdftag/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pkgconfig, meson, vala, ninja
+, gtk3, poppler, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "pdftag";
+  name = "${pname}-${version}";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "arrufat";
+    repo = pname;
+    rev = version;
+    sha256 = "17zk42h0n33b4p8fqlq2riqwcdi8y9m5n0ccydnk6a4x8rli97b3";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson ninja gtk3 poppler wrapGAppsHook ];
+  buildInputs = [ vala ];
+
+  patchPhase = ''substituteInPlace meson.build \
+    --replace "install_dir: '/usr" "install_dir: '$out"
+  '';
+
+  preInstall = "mkdir -p $out/share/licenses/${pname}";
```

Comment:
I think installing licenses is Debianism. Maybe asking upstream about disabling this might be even more important than changing the path.

Reply:
@jtojnar: I filed an issue upstream. Note that in some cases the license text *has* to be distributed alongside the rest as part of the license itself. Installing it in $prefix seems a sane way to go.